doc/source/visualizing/callbacks.rst

```diff
@@ -94,7 +94,7 @@ dataset from AMRVAC :
     import yt
 
     ds = yt.load("amrvac/bw_polar_2D0000.dat")
-    s = yt.plot2d(ds, 'density')
+    s = yt.SlicePlot(ds, axis=2, fields=['density'])
```

Comment:
I think the explicit wart syntax is much better, especially in documentation, but I'm curious: why the change from plot_2d to SlicePlot ?

Reply:
It's simple, I couldn't find plot2d and I didn't even know that `plot_2d` existed :)

Reply:
Oh nice, I'm only seeing the typo now, haha.
That's what code review is for I guess, always can use another pair of eyes !

Reply:
reverted and fixed the typo